app/Http/Controllers/Auth/AuthenticatedSessionController.php

```diff
@@ -16,15 +17,20 @@ class AuthenticatedSessionController extends Controller
      */
     public function create(): View
     {
-        return view('login');
+        if (env('APP_ENV') == 'local'){
```

Comment:
revert the codes you have to set the email, password only in blade.

Reply:
> revert the codes you have to set the email, password only in blade.

refactored as u mentioned please check again @Raza9798 #58